drivers/wifi/nrf700x/zephyr/src/zephyr_wifi_mgmt.c

```diff
@@ -739,129 +787,3 @@ void wifi_nrf_event_proc_twt_sleep_zep(void *vif_ctx,
 	break;
 	}
 }
-
-int wifi_nrf_set_power_save_timeout(const struct device *dev,
-			 struct wifi_ps_timeout_params *ps_timeout_params)
-{
-	enum wifi_nrf_status status = WIFI_NRF_STATUS_FAIL;
-	struct wifi_nrf_ctx_zep *rpu_ctx_zep = NULL;
-	struct wifi_nrf_vif_ctx_zep *vif_ctx_zep = NULL;
-	int ret = -1;
-
-	if (!dev || !ps_timeout_params) {
-		goto out;
-	}
-
-	vif_ctx_zep = dev->data;
-
-	if (!vif_ctx_zep) {
-		LOG_ERR("%s: vif_ctx_zep is NULL\n", __func__);
-		goto out;
-	}
-
-	if (vif_ctx_zep->if_type != NRF_WIFI_IFTYPE_STATION) {
-		LOG_ERR("%s: Operation supported only in STA mode\n",
-			__func__);
-		goto out;
-	}
-
-	rpu_ctx_zep = vif_ctx_zep->rpu_ctx_zep;
-
-	if (!rpu_ctx_zep) {
-		LOG_ERR("%s: rpu_ctx_zep is NULL\n", __func__);
-		goto out;
-	}
-
-	status = wifi_nrf_fmac_set_power_save_timeout(rpu_ctx_zep->rpu_ctx,
-					  vif_ctx_zep->vif_idx,
-					  ps_timeout_params->timeout_ms);
-
-	if (status != WIFI_NRF_STATUS_SUCCESS) {
-		LOG_ERR("%s: wifi_nrf_fmac_ failed\n", __func__);
-		goto out;
-	}
-
-	ret = 0;
-out:
-	return ret;
-}
-
-int wifi_nrf_set_listen_interval(const struct device *dev,
-				 struct wifi_listen_interval_params *params)
-{
-	enum wifi_nrf_status status = WIFI_NRF_STATUS_FAIL;
-	struct wifi_nrf_ctx_zep *rpu_ctx_zep = NULL;
-	struct wifi_nrf_vif_ctx_zep *vif_ctx_zep = NULL;
-	int ret = -1;
-
-	if (!dev || !params) {
-		goto out;
-	}
-
-	vif_ctx_zep = dev->data;
-
-	if (!vif_ctx_zep) {
-		LOG_ERR("%s: vif_ctx_zep is NULL\n", __func__);
-		goto out;
-	}
-
-	rpu_ctx_zep = vif_ctx_zep->rpu_ctx_zep;
-
-	if (!rpu_ctx_zep) {
-		LOG_ERR("%s: rpu_ctx_zep is NULL\n", __func__);
-		goto out;
-	}
-
-	status = wifi_nrf_fmac_set_listen_interval(rpu_ctx_zep->rpu_ctx,
-						   vif_ctx_zep->vif_idx,
-						   params->listen_interval);
-
-	if (status != WIFI_NRF_STATUS_SUCCESS) {
-		LOG_ERR("%s: wifi_nrf_fmac_ failed\n", __func__);
-		goto out;
-	}
-
-	ret = 0;
-out:
-	return ret;
-}
-
-int wifi_nrf_set_ps_wake_interval(const struct device *dev,
```

Comment:
This is a fixup commit, as this function changes in previous commit, so, either fix or squash/fixup these two commits.